hack/vsphere/set_env

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Enabling vSphere cloud provider feature flag in kops.
+export KOPS_FEATURE_FLAGS=+VSphereCloudProvider
+
+# If set, coredns will be used for vsphere cloud provider.
+export VSPHERE_DNS=coredns
```

Comment:
You already have CLI option "--dns=private --vsphere-coredns-server" so do we still need these environment variable?